pyrax/cloudcdn.py

```diff
@@ -57,14 +57,15 @@ def delete_assets(self, url=None, all=False):
 class CloudCDNServiceManager(BaseManager):
 
     def create(self, name, flavor_id, domains, origins,
-               restrictions=None, caching=None):
+               restrictions=None, caching=None, log_delivery=None):
```

Comment:
If that happened, we'd just add a `log_path=None` argument and construct the appropriate body to go along with it.